Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -48,8 +48,8 @@ void ABenchmarkGymGameMode::GenerateTestScenarioLocations()
 	constexpr float RoamRadius = 7500.0f; // Set to half the NetCullDistance
 	{
 		FRandomStream PlayerStream;
-		PlayerStream.Initialize(FCrc::MemCrc32(&ExpectedPlayers, sizeof(ExpectedPlayers))); // Ensure we can do deterministic runs
-		for (int i = 0; i < ExpectedPlayers; i++)
+		PlayerStream.Initialize(FCrc::MemCrc32(&NumPlayers, sizeof(NumPlayers))); // Ensure we can do deterministic runs
```

Comment:
Why this change?

Reply:
Oh I thought NumPlayers previously existed

Reply:
Yeah, I renamed ExpectedPlayers->NumPlayers so most of these changes are that. I updated the tests with ExpectedPlayers instead